src/logic/useScooters.ts

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from 'react'
+import { ScooterOperator, Scooter } from '@entur/sdk'
+
+import service from '../service'
+import { useSettingsContext, Settings } from '../settings'
+import { REFRESH_INTERVAL } from '../constants'
+
+function countScootersByOperator(
+    list: Scooter[] | null,
+): Record<ScooterOperator, Scooter[]> {
+    const operators: Record<ScooterOperator, Scooter[]> = {
+        [ScooterOperator.VOI]: [],
+        [ScooterOperator.TIER]: [],
+        [ScooterOperator.LIME]: [],
+        [ScooterOperator.ZVIPP]: [],
+    }
+    list?.map((scooter) => operators[scooter.operator].push(scooter))
```

Comment:
Bruk `forEach` i staden for map, eller skriv om til å bruke `reduce`. `map` bruker man helst når man vil returnere ei ny liste med ei endring på kvart element.